client/my-sites/promote-post-i2/components/post-item/index.tsx

```diff
@@ -86,22 +88,31 @@ export default function PostItem( { post }: { post: BlazablePost } ) {
 				</div>
 				<div className="post-item__post-data-row post-item__post-data-row-mobile">
 					<div className="post-item__stats-mobile">
-						{ sprintf(
-							// translators: %s is number of post's visitors
-							_n( '%s visitor', '%s visitors', viewCount ),
-							formatNumber( viewCount, true )
-						) }
+						{ post.type !== 'product' || ! isRunningInWooStore
```

Comment:
The price field won't be returned in WPCOM. It will only work in Jetpack Blaze and Woo Blaze.
For this iteration, I think it will be better to encapsulate the SKU/price change only with the `isRunningInWooStore` flag.

The idea will be to not show the SKU/Price if the dashboard is not running in Woo Blaze.

Reply:
Fixed